app/program.js

```diff
@@ -0,0 +1,29 @@
+var OSinfo = require('../modules/OSinfo');
+
+process.stdin.setEncoding('utf-8');
+
+process.stdin.on('readable', function () {
+    var input = process.stdin.read();
+    if (input !== null) {
+        
+        var instruction = input.trim();
+        
+        switch (instruction) {
```

Comment:
same as in previous task, switch outside the if statement, remember that your function should works exactly the same as now